client/layout/guided-tours/docs/API.md

```diff
@@ -0,0 +1,162 @@
+# Guided Tours API
+
+Guided Tours are declared as a tree of JSX components.
+
+
+## Tour
+
+Tour is a React component that declares the top-level of a tour. It defines conditions for starting a tour and contains `<Step>` components as children.
+
+### Props
+
+* `name`: (string) Unique name of tour in camelCase.
+* `version` (string): Version identifier. We use date string like "20161224".
+* `path` (string or array, optional): Use this prop to limit tour only to some path prefix (or more prefixes if array). Example: `[ '/stats', '/settings' ]`
+* `when` (function, optional): This is a redux selector function. Use this to define conditions for the tour to start.
+
+## Step
+
+Step is a React component that defines a single Step of a tour. It is represented as dark box on top of Calypso UI. Step can be positioned in many ways in relation to any DOM node in Calypso that is marked with `[data-tip-target]` attribute.
+
+### Props
+
+* `name`: (string) Unique identifier of the step.
+* `target`: (string, optional) Target which this step belongs to and will be used for positioning. This prop is value is used to look up according `[data-tip-target]` in DOM. If omitted, `<body>` will be used as target. TODO: mention querySelector hack?, is body really default target?
+* `placement`: (string, optional) Placement. Possible values: 'below', 'above', 'beside', 'center', 'middle', 'right'
+* `arrow`: (string, optional) If defined, step will get arrow pointing to a direction. Available: 'top-left', 'top-center', 'top-right',
+'right-top', 'right-middle', 'right-bottom', 'bottom-left', 'bottom-center', 'bottom-right', 'left-top', 'left-middle', 'left-bottom'
+* `style`: (object, optional) Will be used as step's inline style.
```

Comment:
I would add a word of advice, like:
- use sparingly;
- avoid changing the look and feel of Guided Tours;
- useful for minor positioning adjustments, z-index, etc.